build/cmake/DefineOptions.cmake

```diff
@@ -53,6 +53,11 @@ endif ()
 CMAKE_DEPENDENT_OPTION(BUILD_AS3 "Build as3 library" ON
                        "BUILD_LIBRARIES;WITH_AS3;HAVE_COMPC" OFF)
 
+# OpenSSL (used by C++ and C GLib)
```

Comment:
Should this be inside:

    if (WITH_CPP AND WITH_C_GLIB)

or something like that?  If you are building just java, for example, you don't need it.